docs/source/openmrs.rst

```diff
@@ -4,6 +4,133 @@
+    |                         |click the Person birth Date Estimated flag below and add an |
+    |                         |age.                                                        |
+    +-------------------------+------------------------------------------------------------+
+    |Person birthDateEstimated|Click 'Yes' if the age is estimated and add the age in the  |
+    |flag                     |box to the right. Otherwise, click 'No'                     |
+    +-------------------------+------------------------------------------------------------+
+    |Person gender            |Enter the gender. The OpenMRS default is "M" or "F"         |
+    +-------------------------+------------------------------------------------------------+
+    |Person dead flag         |Required by OpenMRS. Choose 'Yes' if the person is dead and |
+    |                         |enter the date they died.                                   |
+    +-------------------------+------------------------------------------------------------+
+    |Cause of death UUID      |UUID of the OpenMRS concept that caused the death.          |
+    |                         |(i.e. 114100AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA - Pneumonia      |
+    +-------------------------+------------------------------------------------------------+
+    |MotechId                 |The ID value from MOTECH that you would like entered into   |
+    |                         |OpenMRS.                                                    |
+    +-------------------------+------------------------------------------------------------+
+    |Location name            |The name of the OpenMRS location where the patient was      |
+    |                         |created. (i.e. Registration Desk)                           |
+    +-------------------------+------------------------------------------------------------+
+    |Patient Identifiers      |The extra patient Identifiers you would like added as       |
+    |                         |defined in the openmrs.identifierTypes property.            |
+    +-------------------------+------------------------------------------------------------+
+
+- Create Encounter
+    This task action allows you to create patient encounters in OpenMRS. The task uses the POST command to the OpenMRS Rest Web Services module. Below is a description of each field in the action.
+
+    +-------------------------+------------------------------------------------------------+
+    |Field                    |Description                                                 |
+    +=========================+============================================================+
+    |Encounter Date           |The date and time the encounter was created.                |
+    +-------------------------+------------------------------------------------------------+
+    |Encounter Type           |The UUID of the encounter type                              |
```

Comment:
Now in our OpenMRS module the encounter type field in create encounter action is a string representing name of encounter type